Userland/Libraries/LibWeb/Layout/TreeBuilder.cpp

```diff
@@ -211,6 +211,22 @@ void TreeBuilder::create_pseudo_element_if_needed(DOM::Element& element, CSS::Se
     if (!pseudo_element_node)
         return;
 
+    auto& style_computer = document.style_computer();
+    
+    if (is<ListItemBox>(*pseudo_element_node)) {
+        auto marker_style = style_computer.compute_style(element, CSS::Selector::PseudoElement::Type::Marker);
```

Comment:
Can you add a FIXME comment here? (Or maybe a `dbgln("FIXME: ...")`) This code computes style for `element::marker`, not `element::pseudo::marker`. That's fine for now (we have zero support for pseudo-elements with pseudo-elements) but a FIXME will make it easier to find later.